docs/pages/pmd/userdocs/cli_reference.md

```diff
@@ -159,7 +159,8 @@ The tool comes with a rather extensive help text, simply running with `--help`!
                description="Path to a file to which report output is written. The file is created if it does not exist. If this option is not specified, the report is rendered to standard output."
     %}
     {% include custom/cli_option_row.html options="--show-suppressed"
-               description="Causes the suppressed rule violations to be added to the report."
+               description="Causes the suppressed rule violations to be added to the report (xml and html formats only)."
+               languages="html, xml"
```

Comment:
This options is not dependent on the language, but on the renderer, that is used (option `--format`). So we shouldn't add the "languages" attribute here.

I guess, this other sentence should be updated as well:
https://github.com/pmd/pmd/blob/1acaedc158e733cdd10ca771be97417c55991e37/docs/pages/pmd/userdocs/pmd_report_formats.md?plain=1#L21

something like: "... and if the format supports showing suppressed violations. Currently only html, summaryhtml and xml show suppressed violations."

We actually have this already here:
https://github.com/pmd/pmd/blob/1acaedc158e733cdd10ca771be97417c55991e37/docs/pages/pmd/userdocs/pmd_report_formats.md?plain=1#L15

> Not all report formats display all information.

Both are from this documentation page: https://docs.pmd-code.org/latest/pmd_userdocs_report_formats.html

Note: for the CSV format, we have this explicitly documented:
https://docs.pmd-code.org/latest/pmd_userdocs_report_formats.html#csv

> Suppressed violations or errors are ignored.